docs/dev/GeoJsonWrite.md

```diff
@@ -13,42 +13,65 @@ permalink: /docs/dev/GeoJsonWrite/
 
 {% highlight mysql %}
 GeoJsonWrite(VARCHAR path, VARCHAR tableName);
-GeoJsonWrite(VARCHAR path, VARCHAR tableName, VARCHAR fileEncoding);
+GeoJsonWrite(VARCHAR path, VARCHAR tableName, 
+             BOOLEAN deleteTable);
+GeoJsonWrite(VARCHAR path, VARCHAR tableName, 
```

Comment:
Encoding has been removed
The signatures are now

GeoJsonWrite(VARCHAR path, VARCHAR tableName);
GeoJsonWrite(VARCHAR path, VARCHAR tableName, 
             BOOLEAN deleteTable);